pkgs/development/libraries/speechd/default.nix

```diff
@@ -1,38 +1,59 @@
-{ fetchurl, lib, stdenv, intltool, libtool, pkgconfig, glib, dotconf, libsndfile
-, libao, python3Packages
-, withEspeak ? false, espeak
+{ stdenv, lib, pkgconfig, fetchurl, buildPythonApplication
+, intltool, itstool, libtool, texinfo, autoreconfHook
+, glib, dotconf, libsndfile
+, pyxdg
+, libao, libpulseaudio, alsaLib
+, withFestival ? false # TODO: , festival-freebsoft-utils
+, withEspeak-ng ? true, espeak-ng, sonic, pcaudiolib
 , withPico ? true, svox
+, withIvona ? false, # TODO: , libdumbtts
 }:
 
-stdenv.mkDerivation rec {
+with lib;
+buildPythonApplication rec {
```

Comment:
This is definitely not a python application, it is a C library.

Reply:
But it worked so well :-(.

spd-conf is a python application, and doesn't work after the requested change.
spd-say and speech-dispatcher work with both approaches.

Reply:
Right, you would also need to revert the previous python wrapping.

Reply:
Done